src/os/exec_linux.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build posix || aix || linux || dragonfly || freebsd || (js && wasm) || netbsd || openbsd || solaris || wasip1
+// +build posix aix linux dragonfly freebsd js,wasm netbsd openbsd solaris wasip1
```

Comment:
I think you can simplify this to `//go:build linux`.
Also, what about MacOS? I don't see why it wouldn't work.